activemodel/lib/active_model/type/model.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  module Type
+    class Model < Value # :nodoc:
+      def initialize(**args)
+        @class_name = args.delete(:class_name)
+        @serializer = args.delete(:serializer) || ActiveSupport::JSON
+        super
```

Comment:
It might be necessary to use this to prevent `unknown keyword` errors.
```suggestion
        super()
``

Reply:
Sounds fair, I'll do the change similarly to the `Collection` type